torch/custom_class.h

```diff
@@ -153,6 +153,26 @@ class class_ {
     return *this;
   }
 
+  template <typename GetterFunc, typename SetterFunc>
+  class_& def_property(
+      std::string name,
+      GetterFunc getterFunc,
+      SetterFunc setterFunc,
+      std::string getter_doc_string = "",
+      std::string setter_doc_string = "") {
+    auto wrapped_getter =
+        detail::wrap_func<CurClass, GetterFunc>(std::move(getterFunc));
+    auto wrapped_setter =
+        detail::wrap_func<CurClass, SetterFunc>(std::move(setterFunc));
+    defineProperty(
```

Comment:
Unless you have plans to reuse this function, you can include the body of `defineProperty` here.

Reply:
sounds good